app/controllers/models_controller.rb

```diff
@@ -6,6 +6,36 @@ class ModelsController < ApplicationController
 
   def index
     @models = resource_base_search_and_page
+    @hosts_counts = Model.hosts_counts
+    @table = @models.map do |m|
```

Comment:
This looks very hacky, why don't we retrieve the data via a separate API call? That would also support pagination etc.?

Reply:
I didn't do so because I was afraid that might be a problem in regard to API versions. However, if it's no a problem, I am going to do it.